src/App.js

```diff
@@ -1,23 +1,108 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import "./App.css";
+import array from "./contacts.json";
 
 function App() {
+  // let startNumber = 0;
+  let maxNumber = 5;
+  let initialArray = array.slice(0, maxNumber);
+  let remainingArray = array.slice(maxNumber, array.length - 1);
+
+  const [contact, setContacts] = useState(initialArray);
+
+  const addContact = () => {
+    if (contact.length === array.length) {
+      return false;
+    }
+    setContacts(() => {
+      const newContact =
+        remainingArray[Math.floor(Math.random() * remainingArray.length)];
```

Comment:
new contact should be unique , it means we need to check if this contact already exist, dont add it to array ;)